qemu/tests/qemu_disk_img_convert.py

```diff
@@ -33,6 +36,28 @@ def convert(self, t_params=None):
         self.trash.append(converted)
         return params
 
+    @error_context.context_aware
+    def compare_test(self, t_params):
+        """
+        Compare images.
+
+        :param t_params: Dictionary with the test parameters
+        """
+        error_context.context("Compare images with default and strict mode",
+                              logging.info)
+        for strict_mode in t_params.objects("compare_mode_list"):
+            rv = None
```

Comment:
1. suggest to use more clearer name instead of rv.
2. move error_context.context here